### ISSUE TITLE
`uv add` resolution failure error includes current project version

### ISSUE THREAD

**Issue by zanieb** [astral-sh/uv#5342]
```
$ uv add 'httpx>9999'
error: Because only httpx<=9999 is available and example==0.1.0 depends on httpx>9999, we can conclude that example==0.1.0 cannot be used.
And because only example==0.1.0 is available and you require example, we can conclude that the requirements are unsatisfiable.
```

It's really weird to include the following:

> only example==0.1.0 is available and you require example

**Comment by charliermarsh** [comment]
Should it even say "you require example"?

**Comment by zanieb** [comment]
Definitely not, that's part of the problem.

**Comment by charliermarsh** [comment]
Even if `example` is a workspace member, but not the root...?

**Comment by zanieb** [comment]
I'm not sure. We should probably have a special-case for workspace members though 😭

**Comment by zanieb** [comment]
Solved in #6092
